pkg/bucketaccessrequest/bucketaccessrequest.go

```diff
@@ -146,17 +162,47 @@ func (b *bucketAccessRequestListener) provisionBucketAccess(ctx context.Context,
 	return nil
 }
 
-func (b *bucketAccessRequestListener) FindBucketAccess(ctx context.Context, bar *v1alpha1.BucketAccessRequest) *v1alpha1.BucketAccess {
+func (b *bucketAccessRequestListener) removeBucketAccess(ctx context.Context, bucketAccessRequest *v1alpha1.BucketAccessRequest) error {
+	bucketaccess := b.FindBucketAccess(ctx, bucketAccessRequest)
+	if bucketaccess == nil {
+		// bucketaccess for this BucketAccessRequest is not found
+		return util.ErrBucketAccessDoesNotExist
```

Comment:
Why is this an error? The BA is gone.

Reply:
This is a know error that we need not retry.